### PR TITLE
fix hard coding api server into watchdog image

### LINKED ISSUE #866: Cluster specific configuration built into image. This image could only be used in one specific cluster.
https://github.com/Microsoft/pai/blob/7c7065d878e64a4a9df2bc92ac24e0b360af7ae1/pai-management/src/watchdog/dockerfile.template#L39

### PR DESCRIPTION
**Pull request by xudifsd**
fix #866

### CODE REVIEW

File: examples/kafka/kafka.json [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-          "beginAt": 0,
-          "portNumber": 1
-        }
-      ]
-    }
-  ]
-}
+{
+  "jobName": "kafka",
+  "image": "openpai/pai.example.kafka",
```
Comment (YanjieGao) [inline examples/kafka/kafka.json]: Why affect this file?

Comment (xudifsd) [reply examples/kafka/kafka.json]: this file's line ending is \r\n, so I use dos2unix to converted this file.

Comment (YanjieGao) [reply examples/kafka/kafka.json]: ok, this is better not include at this PR, may confuse others.

**Comment by xudifsd** [comment]
@YanjieGao done

**Pull request merged**